api/src/pcapi/routes/backoffice/pivots/blueprint.py

```diff
@@ -76,6 +80,17 @@ def create_pivot(name: str) -> utils.BackofficeResponse:
     try:
         can_create_pivot = pivot_context.create_pivot(form)
         if can_create_pivot:
+            venue = offerer_models.Venue.query.filter_by(id=form.venue_id.data[0]).one_or_none()
+            if not venue:
+                flash("Lieu inexistant", "warning")
+                return redirect(url_for(".get_pivots", active_tab=name), code=303)
```

Comment:
il faut déplacer cette partie plus haut dans la fonction (au niveau du `if not form.validate()`) pour éviter de commencer la création de pivot.